src/fosslight_util/_get_downloadable_url.py

```
@@ -540,50 +575,65 @@ def get_latest_package_version(link, pkg_type, oss_name):
+            if group_path and artifact_id:
+                for repo_base in MAVEN_REPOSITORY_BASES:
+                    find_version = _maven_latest_version_from_repo(
+                        repo_base, group_path, artifact_id
+                    )
+                    if find_version:
+                        logger.info(
+                            f'Maven latest version {find_version} from {repo_base}'
+                        )
+                        break
+            if not find_version:
+                maven_response = requests.get(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)